Lib/inspect.py

```diff
@@ -268,7 +268,7 @@ def iscode(object):
         co_kwonlyargcount   number of keyword only arguments (not including ** arg)
         co_lnotab           encoded mapping of line numbers to bytecode indices
         co_name             name with which this code object was defined
-        co_names            tuple of names of local variables
+        co_names            tuple of names of global variables
```

Comment:
This description is currently under the `code` type... The [other descriptions](https://docs.python.org/3.6/library/inspect.html#types-and-members) don't make explicit reference to bytecode (except for `co_consts`) since it is implied by the `code` type. Would be pretty repetitive to add reference to the bytecode in every description. Thoughts?